src/either/either.fork.test.ts

```diff
@@ -135,3 +135,13 @@ test('result don`t updates for not selected argument', async () => {
     ]
   `);
 });
+
+test('do not make infinite loop', async () => {
```

Comment:
I suggest to renamie it as 
```suggestion
test('config shape is  supported', async () => {
```
and update it as such